lib/ui/page/home/page/chat/widget/chat_item.dart

```diff
@@ -245,6 +246,7 @@ class ChatItemWidget extends StatefulWidget {
           onTap: isLocal
               ? null
               : () {
+                  if (disableGallery) return;
```

Comment:
This doesn't disable the tap itself. The tap is possible, it's just its handler returns immediately. Don't you think disabling the gesture would be a cleaner approach?

Reply:
@SleepySquash Yes, but disabling tap behaviour on gesture was closing the context menu when I tap on the image. Will it be okay to do that ?

Reply:
@NikeshNayak, ah, I see, that's ok then. You should add some comments describing code, when something not really trivial happens. I'll continue the review then.